www/heatmap.html

```diff
@@ -0,0 +1,239 @@
+var heatmap_display_delay = 25; // sets a 25 millisecond delay before each new point is displayed
+var heatmap_data_counter = 0;
+var heatmap_data_length = heatmap_data.data.length;
+var heatmap_timer;
+var heatmap_displayed_data = {data:[]}; // starts off empty but clones the data to display point by point
+
+var repeat = true; // Whether to repeat the simulation in a continous loop
+
+// function that initializes the visulaization by repeatedly calling heatmap_display_function()
+function heatmap_trigger_display() { 
+
+    heatmap_timer = setInterval(heatmap_display_function, heatmap_display_delay); // sets the interval between each point displaying
+    heatmap_data_counter++;
+}
+
+function heatmap_display_function() {
+
+    console.log("Starting Simulation");
+    if(heatmap_data_counter == heatmap_data_length) {
+
+        if(repeat){
+
+            console.log("Reset Simulation");
+            var empty_data = {
+                data : {}
+            };
+            heatmap.setData(empty_data);
+            heatmap_displayed_data = {data:[]};
+            heatmap_data_counter = 0;
+        } else {
+            clearInterval(heatmap_timer);
+        }
+
+    }
+
+    heatmap_displayed_data.data.push(heatmap_data.data[heatmap_data_counter]);
+    heatmap.setData(heatmap_displayed_data); // renders the data on the screen
+    heatmap_data_counter++;
+}
+
+setTimeout(function () {heatmap_trigger_display();}, initial_delay); // starts displaying the points after the inital delay time has run out</code></pre>
+
+            <p><br>Save the script as <code class="javascript">heatmap-webgazer.js</code> or download it below, into the same location as the rest of your <code class="javascript">.js</code> files. Import this script into the HTML.</p>
+
+<pre><code class="html">&lt;!-- Script Element --&gt;
+&lt;script src="./js/heatmap.js"&gt;&lt;script&gt; 
+&lt;script src="./js/heatmap-data.js"&gt;&lt;script&gt;
+&lt;script src="./js/heatmap-webgazer.js"&gt;&lt;script&gt; // import the file we just created</code></pre>
+
+        </div>
+
+
+        <div id="Download" class="container pad">
+           <hr class="featurette-divider">
+           <h2 class="text-center">Download</h2>
+           <div class="row">
+               <div class="col-lg-4">
+                    <h4 class="text-center">    
+                            <a href="/js/heatmap-webgazer.js" download>Sample Code</a>
+                    </h4>
+                </div>
+                <div class="col-lg-4">
+                    <h4 class="text-center">    
+                            <a href="/js/heatmap-data.js" download>Sample Data</a>
+                    </h4>
+                </div>
+                <div class="col-lg-4">
+                    <h4 class="text-center">    
+                            <a href="/heatmap-demo.html" download>Sample HTML</a>
```

Comment:
This HTML file won't work for the user unless they also download the WebGazer image used in it too.